src/Stratis.Features.FederatedPeg/Collateral/CollateralChecker.cs

```diff
@@ -158,11 +158,7 @@ private async Task UpdateCollateralInfoAsync(CancellationToken cancellation)
 
             if (addressesToCheck.Count == 0)
             {
-                this.collateralUpdated = true;
```

Comment:
I'm not sure that this is valid as we are checking the number of federation member addresses. So if there is none configured why bother with the collateral check?

Reply:
Because it's not just a collateral check. The `counterChainConsensusTipHeight` variable is also being updated.